server/lib/src/idm/scim.rs

```diff
@@ -548,7 +548,15 @@ impl<'a> IdmServerProxyWriteTransaction<'a> {
         &mut self,
         sse: &'b ScimSyncUpdateEvent,
         changes: &'b ScimSyncRequest,
-    ) -> Result<(Uuid, BTreeSet<String>, BTreeMap<Uuid, &'b ScimEntry>, bool), OperationError> {
+    ) -> Result<
+        (
+            Uuid,
+            BTreeSet<String>,
+            BTreeMap<Uuid, &'b ScimEntryGeneric>,
+            bool,
+        ),
```

Comment:
That's a truly horrifying result type 😄

Reply:
I think it could be changed, but it's been that way for a while.

Reply:
// TODO: less crimes